.eslintrc.js

```diff
@@ -533,5 +534,10 @@ module.exports = {
 		'you-dont-need-lodash-underscore/to-pairs': 'error',
 		'you-dont-need-lodash-underscore/to-upper': 'error',
 		'you-dont-need-lodash-underscore/uniq': 'error',
+
+		// @TODO remove these lines once we fixed the warnings so
+		// they'll become errors for new code added to the codebase
+		'@tanstack/query/exhaustive-deps': 'warn',
+		'@tanstack/query/prefer-query-object-syntax': 'warn',
```

Comment:
I think it's a neat idea to ship these as warnings and change them to errors once all have been resolved 👍